pkg/utils/utils.go

```diff
@@ -49,3 +52,76 @@ func IsZero[C comparable](val C) bool {
 	var zero C
 	return zero == val
 }
+
+// JustError takes a tuple and returns the last entry, the error.
+func JustError(_ interface{}, err error) error {
```

Comment:
Is this used in production?

Reply:
Mostly in tests, but there are a few exceptions like:
- https://github.com/smartcontractkit/chainlink/blob/develop/core/services/pg/lease_lock.go#L181
- https://github.com/smartcontractkit/chainlink/blob/develop/core/cmd/dkgencrypt_keys_commands.go#L27
- https://github.com/smartcontractkit/chainlink/blob/develop/core/cmd/admin_commands.go#L158

Reply:
It seems like overkill that we don't want to encourage but I guess that is out of scope :shrug: